src/main/java/me/glaremasters/guilds/utils/LoggingUtils.java

```diff
@@ -79,12 +78,12 @@ public static void severe(String msg) {
      * Guilds logLogo in console
      */
     public static void logLogo(ConsoleCommandSender sender, Plugin plugin) {
-        sender.sendMessage(ACFBukkitUtil.color("&a  ________ "));
-        sender.sendMessage(ACFBukkitUtil.color("&a /  _____/ "));
-        sender.sendMessage(ACFBukkitUtil.color("&a/   \\  ___ " + "  &3Guilds &8v" + plugin.getDescription().getVersion()));
-        sender.sendMessage(ACFBukkitUtil.color("&a\\    \\_\\  \\" + "  &3Server Version: &8" + plugin.getServer().getVersion()));
-        sender.sendMessage(ACFBukkitUtil.color("&a \\______  /"));
-        sender.sendMessage(ACFBukkitUtil.color("&a        \\/ "));
+        sender.sendMessage(StringUtils.color("&a  ________ "));
+        sender.sendMessage(StringUtils.color("&a /  _____/ "));
+        sender.sendMessage(StringUtils.color("&a/   \\  ___ " + "  &3Guilds &8v" + plugin.getDescription().getVersion()));
+        sender.sendMessage(StringUtils.color("&a\\    \\_\\  \\" + "  &3Server Version: &8" + plugin.getServer().getVersion()));
+        sender.sendMessage(StringUtils.color("&a \\______  /"));
+        sender.sendMessage(StringUtils.color("&a        \\/ "));
```

Comment:
sending messages like this has the possibility to be interrupted by outside actors, consider using a stringbuilder to append all lines, and then send it in a single message

Reply:
(if this was in Kotlin you could use the buildString function ;P)